about.html

```diff
@@ -0,0 +1,138 @@
+        <nav class="mobile-nav">
+          <div class="close-btn">
+              <i class="fa-solid fa-xmark"></i>
+          </div>
+        <ul class="nav-list">
+          <li>
+            <a class="nav-links" href="about.html">About</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">Program</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">Join</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">Sponser</a>
+          </li>
+          <li>
+            <a class="nav-links" href="#">News</a>
+          </li>
+          <li>
+            <a href="#">
+                <button class="mobile-nav-btn">Campaign</button>
+            </a>
+          </li>
+        </ul>
+      </nav>  
+        <div class="menu-btn">
+            <i class="fa-sharp fa-solid fa-bars"></i>
+        </div>
+        <div class="about-title">
+            <img src="./images/about_title.png" alt="about title">
+        </div>
+        <div class="red-line"></div>
+        <p class="about-paragraph">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusantium molestiae tenetur vero consectetur voluptatum sunt debitis nisi, 
+            nemo ullam reprehenderit, sit rem laborum aliquam cupiditate excepturi dolor autem, velit quasi?
+        </p>
+        <p class="contact">
+          Please contact us via email for any further question about CC Global
+          Submit 2015!<br> <a href="#">ccsummit@gmail.com</a>
+        </p>
+    </header>
+    <hr>
+    <!--Logo Section-->
+    <section class="log-section">
+        <h2 class="logo-title">Creative Commons <br class="logo-br"> Global Submit 2015 logo</h2>
+        <div class="red-line"></div>
+        <p class="logo-paragraph">
+            The logo of CC Global Summit 2015 was decided <br class="logo-br"> through the logo competition form 2.April to 9.May.
+        </p>
+        <img class="logo" src="./images/about-logo.png" alt="My logo">
+    </section>
+    <hr>
+    <!--Photos Section-->
+    <section class="photos-section">
+        <h2 class="Photos-title">See the past CC Global Summits</h2>
+        <div class="red-line"></div>
+        <p class="Photos-paragraph">
+            Take a look at the last two CC Global Summits which took place in Dubai and Doha.
+        </p>
+        <div class="previous-photos">
+            <img src="./images/about_2013.png" alt="2013 photo">
+            <img src="./images/about_2011.png" alt="2011 photo">
+        </div>
+    </section>
+    <!--Footer Section-->
+    <section class="footer">
+       <div class="partner-img">
+         <img class="footer-logo" src="./images/partner.PNG" alt="partner image">
+       </div>
+       <div class="footer-img">
+        <img src="./images/about-footer.PNG" alt="Footer Image">
+       </div>
+       <div class="about-desktop-footer">
+        <img src="./images/about-desk-footer.PNG" alt="">
+       </div>
+    </section>
```

Comment:
- [ ] The footer section would be better placed in a proper `<footer>` tag. Kindly change that for better HTML semantics. 
- [ ] The footer should have a logo on the left and text on the right, like the image you added, but it should not be an `<img>` tag. You can use the `<img>` tag for the logo on the left, but write the text in the `<p>` tags on the right. Please apply those changes and try to keep the same style and sizing as the template.
 - [ ] Please create a partners section for the mobile version above the footer with a "Partner" heading, a divider and 5 partners like the template.

Reply:
Done